.ebextensions/yarn.config

```
@@ -0,0 +1,8 @@
+packages:
+  rpm:
+    nodesource: https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm
+container_commands:
+  01_upgrade_nodejs:
+    command: yum -y install nodejs
+  02_install_yarn:
+    command: npm install --global yarn
```

```suggestion
packages:
  rpm:
    nodesource: https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm
  yum:
    nodejs: []

commands:
  install_yarn:
    command: npm install --global yarn
```

Pretty sure this will work. Maybe not a big difference but seems cleaner.